enterprise/admin/views.py

```diff
@@ -723,6 +776,33 @@ def get_pending_enrollment_message(cls, pending_users, enrolled_in):
             )
         )
 
+    @classmethod
+    def get_failed_ecommerce_order_message(cls, users, course_id):
+        """
+        Create message for the users who were enrolled in a course but ecommerce order was not created successfully.
+
+        Args:
+            users: An iterable of users who were not successfully enrolled
+            course_id (str): A string identifier for the course
+
+        Returns:
+        tuple: A 2-tuple containing a message type and message text
+        """
+        failed_emails = [user.email for user in users]
+        count = len(failed_emails)
+
+        return (
+            'error',
+            ungettext(
+                'Learner {emails} is enrolled in {course_id} but ecommerce order creation failed',
+                'The following learners are enrolled in {course_id} but ecommerce order creation failed : {emails}',
+                count
```

Comment:
We can directly call `len(failed_emails)` ... we are not re-using it for now at other place.